lib/repl.js

```diff
@@ -1345,6 +1334,15 @@ function isCodeRecoverable(code) {
   return stringLiteral ? lastChar === '\\' : isBlockComment;
 }
 
+function _parseREPLKeyword(repl, keyword, rest) {
```

Comment:
@TimothyGu sure, but how is that any better than what is there now? Wouldn't that mean doing something like this?

```js
REPLServer.prototype.parseREPLKeyword = util.deprecate(
  function(keyword, rest) {
    return _parseREPLKeyword.call(this, keyword, rest);
  }, 'REPLServer.parseREPLKeyword() is deprecated', 'DEP0XX');
```
I don't see how this improves the code. Is `.call()` inherently faster?